vagrant/3scale.csv

```diff
@@ -0,0 +1,1000 @@
+"aa20ba69-a86e-4294-981b-41cd258aff47.benchmark","/11?app_id=ef33ece14f9cd7ca&app_key=cbf06921d90fcd2b"
```

Comment:
We do not want this file commited

Reply:
Why not ? It's a dummy data that resolves to the Vagrant VM backend api machine used when smoke testing. The dummy traffic file uses the same host name for the injector agent to establish a connection to a backend api host.

Reply:
This dummy data file has been removed.